superset/app.py

```diff
@@ -277,7 +277,9 @@ def init_views(self) -> None:
         appbuilder.add_view_no_menu(TableModelView)
         appbuilder.add_view_no_menu(TableSchemaView)
         appbuilder.add_view_no_menu(TabStateView)
-        appbuilder.add_view_no_menu(TagView)
+
+        if feature_flag_manager.is_feature_enabled("TAGGING_ENDPOINTS"):
```

Comment:
Would be great to add a test for this, although it's not trivial to do so with our current testing setup :,(